src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,27 @@
+import classNames from 'classnames';
+
+type Props = {
+  showError: boolean,
+  error: string,
+  onErrorClose: () => void,
+};
+
+export const ErrorNotification: React.FC<Props> = ({
+  showError,
+  error,
+  onErrorClose
+}) => (
+  <div
+    className={classNames(
+      'notification is-danger is-light has-text-weight-normal',
+      { hidden: !showError },
+    )}
+  >
+    <button
+      type="button"
```

Comment:
you can add `aria-label` attribute to resolve the lint error 
<img width="410" alt="Screenshot 2023-04-07 at 20 01 47" src="https://user-images.githubusercontent.com/32853034/230655908-8732b34f-ed94-4e19-afff-1e2624864e6a.png">